helm/azure-operator/templates/configmap.yaml

```diff
@@ -75,7 +81,6 @@ data:
         {{- end }}
       kubernetes:
         incluster: true
-      registry:
```

Comment:
That's not what I expected when we talked about mirroring the registry 


![image](https://user-images.githubusercontent.com/627038/89195663-5e822c00-d5a9-11ea-9567-f01c3922659c.png)